tidb-lightning/monitor-tidb-lightning.md

```diff
@@ -23,24 +23,14 @@ pprof-port = 8289
 ...
 ```
 
-and in `tikv-importer.toml`:
-
-```toml
-# Listening address of the status server.
-status-server-address = '0.0.0.0:8286'
-```
-
 You need to configure Prometheus to make it discover the servers. For instance, you can directly add the server address to the `scrape_configs` section:
 
 ```yaml
 ...
 scrape_configs:
-  - job_name: 'lightning'
+  - job_name: 'tidb-lightning'
     static_configs:
       - targets: ['192.168.20.10:8289']
-  - job_name: 'tikv-importer'
-    static_configs:
-      - targets: ['192.168.20.9:8286']
```

Comment:
Should not be deleted. Be careful when you delete content that is not from the original PRs. @TomShawn

Reply:
Thanks! Add back in https://github.com/pingcap/docs/pull/3660/commits/ff53d99da94806f17563b5059d03fcdee83e3fd5.